workflow/controller/controller.go

```diff
@@ -229,6 +229,7 @@ func (wfc *WorkflowController) Run(ctx context.Context, wfWorkers, workflowTTLWo
 				logCtx.Info("started leading")
 				ctx, cancel = context.WithCancel(ctx)
 
+				// TODO JPZ13 12/24/2020 - refactor to use untilwithcontext
```

Comment:
Should this be an issue instead? I've found that `// TODOs` in the codebase just stay there and nothing ever gets done. Why not open a new issue and assign to yourself? Then we can remove this comment

Reply:
Good catch. I'll remove the `TODO` and do a `grep` for any others that I put in and file issues instead